airbyte-config/models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -54,6 +54,12 @@
 
   WorkerEnvironment getWorkerEnvironment();
 
+  long getMinimumWorkspaceRetentionDays();
```

Comment:
Can you create a data class instead of 3 different methods? `WorkspaceRetentionConfig` or smth like that